lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -57,6 +57,21 @@ def stub_email_alert_api_has_subscription(id, frequency, optional_overrides = {}
           .to_return(status: 200, body: response)
       end
 
+      # Stubs the API responses as if each subscription happened in the order they are passed
+      def stub_email_alert_api_has_subscriptions(*subscriptions)
```

Comment:
I think you can justify expanding the comment on the method to clarify what a `subscriptions` object looks like, and how the method should be called. 
There are some examples in the publishing-api test helpers: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb